java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -2892,6 +3329,12 @@ public static ColumnVector boolFromBytes(byte... values) {
     return build(DType.BOOL8, values.length, (b) -> b.appendArray(values));
   }
 
+  public static<T> ColumnVector fromLists(HostColumnVector.ColumnBuilder.DataType dataType, List<T>... lists) {
```

Comment:
Could we add in some java docs to say that this is unstable?  I just don't want people getting sued to this API if we think it is likely to change in later releases.